plugins/ssl/cert_validity

```diff
@@ -0,0 +1,65 @@
+#!/bin/bash
+#
+# check the certificate validity of your certfificates
+#
+# andreas perhab - andreas.perhab@wt-io-it.at
+# https://www.wt-io-it.at/
+#
+# License: GPLv2
+#
+# Installing: Add list of your certificates prefixed by the type in munin plugin-conf.d
+#
+# For openvpn ca.crt and crl.pem
+# [cert_validity]
+# user root
+# env.CERTS crl:/etc/openvpn/easy-rsa/keys/crl.pem x509:/etc/openvpn/easy-rsa/keys/ca.crt
+#
+# For letsencrypt certficates
+# [cert_validity]
+# user root
+# env.CERTS x509:/etc/letsencrypt/live/domain1.example.com/cert.pem x509:/etc/letsencrypt/live/domain2.example.com/cert.pem
+#
+# Dependencies: openssl
+#
+# Parameters:
+#
+#  config
+
+
+if [ "$1" == "config" ] ; then
```

Comment:
In case of switching to `sh`: use `=` instead of `==`.

Reply:
switched to sh and change the `==` to `=`